index.css

```diff
@@ -0,0 +1,267 @@
+*{
+    margin: 0; 
+    padding: 0;
+} 
+a{
+    text-decoration: none;
+    color: #696969;
+}
+a:visited{
+    text-decoration: none;
+    color: #696969;
+}
+
+/* for travel page */
+h1{
+   position: relative; 
+   z-index: 1;
+   top: 100px;
+   left: 200px;
+}
+
+/* Header Styling */
+
+header{
+    position: fixed; 
+    display: flex; 
+    justify-content: space-around;
+    background-color: #f2f2f2;
+    padding: 30px 0; 
+    z-index: 1;
+    width: 100%; 
+}
+nav{
+    width: 25%;
+}
+.header-nav-items{
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+} 
+
+.header-nav-item{
+    color: #696969; 
+    font-size: 13px;
+    font-family: 'Open Sans', sans-serif; 
+}
+.icons-container{
+    width: 10%;
+    color: #696969;
+}
+.icons-container ul{
+    display: flex; 
+    list-style-type: none;
+    justify-content: space-around; 
+}
+
+
+/* Big Image Section Styling */
+.landing-img{
+    width: 100%; 
+    height: 550px;
+}
+.landing-container{
+    position: relative;
+    margin-bottom: 50px;
+} 
+.content{
+    position: absolute;
+    top: 50%; 
+    left: 50%; 
+    transform: translate(-50%,-50%); 
+    color: white;
+    font-family: 'Open Sans', sans-serif;
+}
+.content span{
+    display: block;
+    text-align: center;
+}
+
+.content span:nth-child(1),.content span:nth-child(3){
+    font-size: 18px;
+}
+.content span:nth-child(2){
+    font-size: 70px;
+    font-weight: bold;
+}
+
+/* Tiles Section Styling */
+.categories-container{
+    display: flex; 
+    flex-direction: column;
+    align-items: center;
+}
+.description{
+    font-size: 15px; 
+    color: #696969; 
+    text-align: center;
+    line-height: 1.2rem;
+    width: 40%;
+}
+.category-contanier{
+    position: relative;
+    margin-left: 20px; 
+}
+.category-image{
+    width: 313px; 
+    height: 260px;
+}
+
+.category-contanier button {
+    position: absolute; 
+    top: 50%; 
+    left: 50%; 
+    transform: translate(-50%,-50%);
+    cursor: pointer;
+}
+
+.categories{
+    display: flex;
+    margin-top: 40px;
+} 
+
+.category-button{
+    width: 156px; 
+    height: 56px;
+    background-color: #fff;
+    font-size: 24px; 
+    color: #2573DA;
+    border: none;
+    outline: none;
+}
```

Comment:
you can put the styles that you used for `category-contanier button` here. since they are for the same element.

Reply:
You're ABSOLUTLY right. I added the class "category-button" and I forgot to refactor. 
I'm on it now.